File: enode/src/main/java/com/enodeframework/messaging/MessageHandlerData.java
package com.enodeframework.messaging;

import com.enodeframework.infrastructure.IObjectProxy;

import java.util.ArrayList;
import java.util.List;

/**
 * @author anruence@gmail.com
 */
public class MessageHandlerData<T extends IObjectProxy> {
    public List<T> AllHandlers = new ArrayList<>();
    public List<T> ListHandlers = new ArrayList<>();
    public List<T> QueuedHandlers = new ArrayList<>();
}

File: enode/src/main/java/org/enodeframework/messaging/MessageHandlerData.kt
package org.enodeframework.messaging

import org.enodeframework.infrastructure.ObjectProxy

/**
 * @author anruence@gmail.com
 */
class MessageHandlerData<T : ObjectProxy> {
    var allHandlers: List<T> = ArrayList()
    var listHandlers: List<T> = ArrayList()
    var queuedHandlers: List<T> = ArrayList()
}